README.md

```diff
@@ -84,7 +74,7 @@ chwifi -r home
 chwifi restart work
 ```
 
-To update recognised profiles, pass either `-u` or `update`.
+To update recognised profiles, pass either `-u` or `update`. This should be done whenever a profile is added or removed from the network-manager.
```

Comment:
```suggestion
To update recognised profiles, pass either `-u` or `update`. This should be done whenever a profile is added or removed.
```

Note that `wifi-menu` is not a network-manager but just a tool. So to be more accurate; let's phrase it as such.